pages/common/git-push.md

```diff
@@ -17,3 +17,7 @@
 - Remove remote branches which don't exist locally
 
 `git push --prune {{REMOTE-NAME}}`
+
+- Publish tags
```

Comment:
I suppose any `push` operation could be considered "publishing". I'd suggest either using the term in all the descriptions or in none, otherwise it seem as if this does something different.